assembly/__tests__/runtime/runtime.spec.ts

```diff
@@ -374,11 +374,28 @@ describe("context", () => {
           expect(context.accountBalance).toStrictEqual(u128.from(6), "Updating the attached deposit should update the account balance");
         });
     });
-      
+  
+  describe("Find the block_timestamp", () => {
+    it("should be the default", () => {
+      let timestamp = context.blockTimestamp;
+      expect(timestamp).toBe(42)
+    });
+
+    it("should be the updatable", () => {
+      Context.setBlock_timestamp(84);
+      let timestamp = context.blockTimestamp;
+      expect(timestamp).toBe(84);
+    });
+
+  });
       
   it("should be editable", () => {
     Context.setCurrent_account_id("contractaccount");
     expect(context.contractName).toBe("contractaccount", "Wrong contract name");
+    Context.setSigner_account_id("signeraccount");
+    expect(context.sender).toBe("signeraccount", "Wrong signer account");
+    // Context.setSigner_account_pk("public-key-as-string");
+    // expect(context.senderPublicKey).toBe("public-key-as-string", "Wrong public key");

```

Comment:
this blows up if i try to `setSigner_account_pk` here

Reply:
Yeah needs to be base58 encoded.

Reply:
still not clear on this.  but also, wtf?  where is this coming from?

```
expect(context.senderPublicKey).toBe("The secret is to use boxed mix", "Wrong public key");
``